packages/dataviews/src/dataform-controls/datetime.tsx

```diff
@@ -6,17 +6,35 @@ import {
 	TimePicker,
 	VisuallyHidden,
 	SelectControl,
+	Button,
 	__experimentalNumberControl as NumberControl,
 	__experimentalHStack as HStack,
+	__experimentalVStack as VStack,
 } from '@wordpress/components';
-import { useCallback } from '@wordpress/element';
+import { useCallback, useMemo, useState } from '@wordpress/element';
 import { __ } from '@wordpress/i18n';
 
+/**
+ * External dependencies
+ */
+import { DateCalendar, DateRangeCalendar } from '@automattic/ui';
```

Comment:
Whether we should use `automattic/ui` or not is an open question for me (see related conversation about [validation](https://github.com/Automattic/wp-calypso/pull/104145#discussion_r2155039209)). Engaging @youknowriad @aduth here in case they have thoughts to share.

Reply:
I think if the intention is to backport `DateCalendar` and `DateRangeCalendar` to a package which is not Automattic-specific (i.e. a `@wordpress/...` package), then perhaps it's low risk to use them here in the short term?

cc @ciampo if you have thoughts whether that's the expectation.